README.md

```diff
@@ -121,20 +121,25 @@ experimental. As such all cryptography is assumed to be insecure.**
 
 This code-base is still very much under-development and does not provide any
 user-facing functionality. For developers and researchers, there are several
-tests and benchmarks which may be of interest.
-
-To run tests, use:
-
-```
-$ cargo test --all
-```
-
-To run benchmarks, use:
+tests and benchmarks which may be of interest. Few basic setups needed before starting like:
 
+   1. Install [rustup](https://rustup.rs/).  It's a toolchain manager for Rust (Linux | macos | Windows). For installation run the below command in your terminal
+   ```
+        $ curl https://sh.rustup.rs -sSf | sh
 ```
-$ cargo bench --all
+   2. To configure your current shell run
+  
+   ```
+        $ source $HOME/.cargo/env
```

Comment:
This does not exist for me if I installed a different way rather than using the one-liner. I had to add `~/.cargo/bin/` to my `$PATH` with:
`export PATH=$PATH:~/.cargo/bin`
Then I had to run `rustup install stable` and `rustup default stable` in order to download & select the rust setup.

We should make sure the instructions are generic enough to help people, even if they choose to install rustup from a different source (such as from the arch repositories, as in my case).

Reply:
@spble  Should I put one liner cmd ?

Reply:
I think this is fine as it is, I'm going to resolve this in the name of expediting a solution.

We can always update this later if people start having issues. Arch people can figure it out themselves for now -- being on Arch I figure they like it that way :)